docs/file_conventions.rst

```diff
@@ -38,3 +39,46 @@ each of these files).
    width columns. This file format is used by MAGICC6 but has been
    deprecated for MAGICC7 in favour of ``*EMIS.IN`` and ``*.SCEN7``
    files.
+
+
+Namelists
+---------
+
+At the top of each input file there is a namelist called ``thisfile_specifications``. It looks something like this
+
+.. code:: Fortran
+
+   &THISFILE_SPECIFICATIONS
+    THISFILE_DATACOLUMNS    =    7  ,
+    THISFILE_DATAROWS       =  266  ,
+    THISFILE_FIRSTYEAR      = 1750  ,
+    THISFILE_LASTYEAR       = 2015  ,
+    THISFILE_ANNUALSTEPS    =    1  ,
+    THISFILE_FIRSTDATAROW   =   99  ,
+    THISFILE_UNITS          = "kt"  ,
+    THISFILE_DATTYPE        = "REGIONDATA"  ,
+    THISFILE_REGIONMODE     = "RCPPLUSBUNKERS"  ,
+   /
+
+We summarise the meaning of these flags below:
```

Comment:
Terrific!

Reply:
Yes, hopefully one day it can be deleted as the input files will be self describing enough to not need it